middleware/cors.js

```diff
@@ -0,0 +1,75 @@
+const cspConfig = require('../config/http-headers').content_security_policy;
```

Comment:
use object destructoring and CamelCase instead

Reply:
ok